content/using/install.md

```diff
@@ -57,23 +53,13 @@ sudo setcap 'cap_net_bind_service=+ep' /path/to/urbit
 ```
 (Where `urbit` is the urbit executable downloaded with `wget` prior)
 
-### Windows
-
-> Please note that this method of installing Urbit is experimental, and we may not be able to assist you if you encounter issues related to WSL 2.
+Now, you either want to boot either a comet or a planet.
```

Comment:
There should be a new header somewhere around here to indicate the end of the "here are the specific actions you need to download Urbit" section.